praw/reddit.py

```diff
@@ -720,8 +748,15 @@ def request(
             of the request (default: None).
         :param files: Dictionary, filename to file (like) object mapping
             (default: None).
+        :param json: JSON-serializable object to send in the body
+            of the request with a Content-Type header of application/json
+            (default: None). If ``json`` is provided, ``data`` should not be.
 
         """
+        if data and json:
+            raise ClientException(
+                "At most one of `data` and `json` is " "supported."
```

Comment:
The strings can be combined.

```suggestion
                "At most one of `data` and `json` is supported."
```